### ISSUE TITLE
No shards written

### ISSUE THREAD

**Issue by silaxe** [Storj/storjshare-cli#145]
### Package Versions

Replace the values below using the output from `storjshare --version`.

```
Storjshare 5.0.1
Core 2.0.7
Protocol 0.7.1

```

Replace the values below using the output from `node --version`.

```
v4.4.5
```

### Expected Behavior

farmer.db folder supposed to be filled with shards.

### Actual Behavior

I have been running storjshare-cli on a raspberry pi for a week now. However, despite a couple of encouraging messages in my logs (PUBLISH, OFFER) not a single shard was written in the farmer.db folder. I just noticed that a single .ldb file is written at the launch of the program.

Here is an extract of my log file with a typical recurring message:

```
{"level":"debug","message":"updating contact {\"protocol\":\"0.7.1\",\"address\":\"31.201.175.2\",\"port\":4000,\"nodeID\":\"3fa7a524f138f9dd8092e3572c52b6544567f26d\",\"lastSeen\":1470504427788}","timestamp":"2016-08-06T17:27:07.797Z"}
{"level":"debug","message":"contact already in bucket, moving to tail","timestamp":"2016-08-06T17:27:07.800Z"}
{"level":"info","message":"replying to message to 061bd2989737782181e23f1a82bf74f1f14f4649","timestamp":"2016-08-06T17:27:07.830Z"}
{"level":"debug","message":"not waiting on callback for message 061bd2989737782181e23f1a82bf74f1f14f4649","timestamp":"2016-08-06T17:27:18.283Z"}
{"level":"info","message":"received valid message from {\"protocol\":\"0.7.1\",\"address\":\"31.201.175.2\",\"port\":4000,\"nodeID\":\"3fa7a524f138f9dd8092e3572c52b6544567f26d\",\"lastSeen\":1470504438306}","timestamp":"2016-08-06T17:27:18.307Z"}
```

And my config.json file:

```
{
  "keypath": "/home/pi/.storjshare/id_ecdsa",
  "address": "16bBiJKzBSnawq1n2KrY12SAmBemPmEcwf",
  "storage": {
    "path": "/home/pi/.storjshare",
    "size": 500,
    "unit": "GB"
  },
  "network": {
    "address": "mypublicip",
    "port": 8000,
    "seeds": [
],
    "opcodes": [
      "0f01020202",
      "0f02020202",
      "0f03020202"
    ],
    "forward": true,
    "tunnels": 3,
    "tunnelport": 8001,
    "gateways": {
      "min": 8002,
      "max": 8004
    },
    "concurrency": 2
  },
  "telemetry": {
    "service": "https://status.storj.io",
    "enabled": true
  },
  "loglevel": 4
}
```

Any help would be greatly appreciated !

Thanks

**Comment by bookchin** [comment]
Unfortunately I don't have enough information to say what the issue might be. Have a look at https://storj.readme.io/docs/storjshare-troubleshooting-guide and if you find the cause or discover a bug, please open another issue.
